app/views/admin/content/_merge.html.erb

```diff
@@ -0,0 +1,13 @@
+<div>
+  <h3>Merge Articles</h3>
+  <form action="/admin/content/merge/<%= @article.id %>" method="get">
+    <div class='clearfix'>
+      <label for="merge_with" style="width: 55px; margin-right: 5px;"><%= _("Article ID")%></label>
```

Comment:
I wanted to but I couldn't figure out where I was supposed to put CSS definitions! This would have been something I would have asked my team about best practice I think...